lib/matplotlib/backends/_backend_pdf_ps.py

```diff
@@ -16,6 +19,35 @@ def _cached_get_afm_from_fname(fname):
         return AFM(fh)
 
 
+def get_glyphs_subset(fontfile, characters):
```

Comment:
I suggest adding a unit test for this function: call the function with some font and some characters, read the output using FT2Font, check that the characters you want are in there and (some) characters you don't want are not in there.

Reply:
Added a bunch of tests (not sure about what we _don't_ want in there though